src/build.py

```diff
@@ -234,6 +241,26 @@ def Tar(directory, print_content=False):
   return tar
 
 
+def Zip(directory, print_content=False):
+  assert os.path.isdir(directory), 'Must be a directory'
+  (dir, basename) = os.path.split(directory)
+  archive = os.path.join(dir, basename + '.zip')
+  print 'Creating zip archive', archive
+  with zipfile.ZipFile(archive, 'w', zipfile.ZIP_DEFLATED) as zip:
+    def Add(path, zippath):
```

Comment:
Forgot about `os.walk`. Looking at the result, I'm not sure which I prefer but walk is shorter at least?